cadastro-refeicao/Models/Refeicao.swift

```diff
@@ -7,24 +7,53 @@
 
 import UIKit
 
-class Refeicao: NSObject {
+class Refeicao: NSObject, NSCoding {
+    
+    //MARK: - Atributos
     
     let nome: String
     let felicidade: Int
     var itens: Array<Item> = []
     
-
+    //MARK: - init
+    
     init(nome: String, felicidade: Int, itens: [Item] = []) {
         self.nome = nome
         self.felicidade = felicidade
     }
     
+    //MARK: - NSCoding
+    
+    func encode(with acoder: NSCoder) {
+        acoder.encode(nome, forKey: "nome")
+        acoder.encode(felicidade, forKey: "felicidade")
+        acoder.encode(itens, forKey: "itens")
+    }
+    
+    required init?(coder: NSCoder) {
+        nome = coder.decodeObject(forKey: "nome") as! String
```

Comment:
Mesmo caso, ao forçar o `unwrap` pode vir a dar um erro fazendo com que o app feche, use a coalescência.